Lib/socket.py

```diff
@@ -67,6 +67,7 @@
            "has_dualstack_ipv6", "AddressFamily", "SocketKind"]
 __all__.extend(os._get_exports_list(_socket))
 
+
```

Comment:
No no no, don't remove the `__getattr__` entirely, just don't emit the deprecation there. Now, simply importing socket will cause the deprecation warning.